content/rust/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Data Types in Rust'
+Description: 'An overview of data types in Rust, including scalar and compound types.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Rust'
+  - 'Data Types'
+CatalogContent:
+  - 'rust-for-programmers'
+  - 'paths/computer-science'
+---
+
+In Rust, data types classify the types of values that can be used and manipulated within a program. Rust's type system ensures that data types are known at compile time, which helps catch errors early.
+
+## Scalar Types
+
+Rust has four primary scalar types:
```

Comment:
We can add a line here before the "Rust has four primary scalar types", that the scalar type represents a single value.